### PR TITLE
Create Quiz Backend

### PR DESCRIPTION
**Pull request by tigerkh3**
Backend working properly. Data send from client is being received and inputted into Postgres Database.

Tested for MC / TF Quizzes, Only TF Quizzes, and Only MC Quizzes.

Added body-parser for incoming requests. All routes/models using proper exports and matching file structure.

### CODE REVIEW

File: server/database/index.js [hunk 1]
```diff
@@ -1 +1 @@
-module.exports = require('./config/index');
\ No newline at end of file
+module.exports = require('../config/index');
```
Comment (bennyv8) [inline server/database/index.js:1]: double checked, file path is './config/index'

File: server/index.js [hunk 2]
```diff
@@ -2,16 +2,17 @@ require("dotenv").config();
...
-const { createQuiz } = require ('./routes')
+const { create } = require('./routes/index.js')
+const bodyParser = require('body-parser')
 
 // =============================================
 //                Middleware
 // =============================================
 app.use(express.json());
-
+app.use(bodyParser.json());
```
Comment (bennyv8) [inline server/index.js:14]: Was there issues using express.json?

**Review by bennyv8 — APPROVED** [review]
Notes:
- create quiz logic to API, solid
- recently merged a PR to connect to API via process.env.REACT_APP_API_URL for future use

File: server/index.js (hunk 2 above)
Comment (tigerkh3) [reply server/index.js:15]: Wasn't 100% sure if express.json was working when debugging the initial connection, was just an attempt to test if it work, but it worked after. I can test without it bodyparser and remove it if express.json works as intended with logic.

**Pull request merged**